AntennaTracker/servos.cpp

```diff
@@ -7,18 +7,19 @@
 // init_servos - initialises the servos
 void Tracker::init_servos()
 {
-    // setup antenna control PWM channels
-    channel_yaw.set_angle(g.yaw_range * 100/2);        // yaw range is +/- (YAW_RANGE parameter/2) converted to centi-degrees
-    channel_pitch.set_angle((-g.pitch_min+g.pitch_max) * 100/2);    // pitch range is +/- (PITCH_MIN/MAX parameters/2) converted to centi-degrees
+    SRV_Channels::set_default_function(CH_YAW, SRV_Channel::k_steering);
```

Comment:
I see what's happening here, but the fact that the channels in the next few lines of code are now referred to as "steering" and "elevator" where they were previously called "yaw" and "pitch" does make the code a bit less clear than it was.   afterall, an antenna tracker has neither steering nor elevator.